packages/data-manipulation/html/src/css.ts

```
@@ -0,0 +1,87 @@
+ * Supports both string and object inputs.
+ * @param value The CSS string or object to process
+ * @param escape If true, escapes CSS. If false, returns CSS as-is.
+ * @returns The processed CSS string
+ * @example
+ * css(':where(.UnderlineNav-actions ul) { animation: 1ms rgh-selector-observer; }', true)
+ * @example
+ * css({ padding: "1px" }, true)
+ * @example
+ * css({ margin: 20, padding: 10 }, false)
+ */
+function css(value: string | FlexibleCSSProperties | Properties, escape?: boolean): string;
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)